app/workers/rapid_ready_for_decision/disability_compensation_job.rb

```diff
@@ -59,13 +59,16 @@ def medications(client)
     def send_fast_track_engineer_email_for_testing(form526_submission_id, error_message, backtrace)
       # TODO: This should be removed once we have basic metrics
       # on this feature and the visibility is imporved.
-      body = "A claim just errored on the #{Rails.env} environment " \
-             "with submission id: #{form526_submission_id} and job_id #{jid}." \
-             "The error was: #{error_message}. The backtrace was:\n #{backtrace.join(",\n ")}"
+      body = <<~BODY
+        A claim errored in the #{Rails.env} environment \
```

Comment:
If you want to get an accurate response for the environment, you will need to use Settings.vsp_environment instead of Rails.env. I believe our dev, staging, and sandbox environments all have Rails.env set to 'production'.

Reply:
Thanks for catching this. I _think_ we probably want to catch errors in both Staging and Production. Dev wont send an email so 🤷  but it would probably be best to explicitly exclude it.

Reply:
Updated.